src/main/java/sirius/biz/web/DisasterModeInfo.java

```diff
@@ -121,7 +122,7 @@ private void updateFromRedis() {
                 displayMessageStartTime =
                         NLS.parseMachineString(LocalDateTime.class, db.get(REDIS_DISASTER_MESSAGE_START));
                 maintenancePreviewMessage = db.get(REDIS_DISASTER_PREVIEW_MESSAGE);
-                lockScope = NLS.parseMachineString(Boolean.class, db.get(REDIS_DISASTER_LOCKED));
+                lockScope = Value.of(db.get(REDIS_DISASTER_LOCKED)).asBoolean();
                 lockStartTime = NLS.parseMachineString(LocalDateTime.class, db.get(REDIS_DISASTER_LOCK_START));
                 maintenanceLockMessage = db.get(REDIS_DISASTER_LOCK_MESSAGE);
```

Comment:
Macht es Sinn, die anderen NLS... auch durch Value.of o.ä. zu ersetzen? Oder gibt das Value nicht her?